### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by IvanGuculyak**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,23 @@
...
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    new_list = [Person(person["name"], person["age"]) for person in people]
+
+    for person in people:
+        if "wife" in person and person["wife"]:
```
Comment (ArsenPidhoretskyi) [inline app/main.py:14]: use: `person.get("wife") is not None`

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,23 @@
...
+            Person.people[person["name"]].wife = Person.people[
+                person["wife"]
+            ]
+        if "husband" in person and person["husband"]:
```
Comment (ArsenPidhoretskyi) [inline app/main.py:18]: use: `person.get("husband") is not None`

File: app/main.py [hunk 3]
```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```
Comment (ArsenPidhoretskyi) [inline app/main.py:2]: Add type hinting
HINT: use `from __future__ import annotations`

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

**Review by olena-hul — APPROVED** [review]

**Comment by olena-hul** [comment]
Good job!

**Pull request closed without merging**